build/build.go

```diff
@@ -208,6 +249,9 @@ func Build(ctx context.Context, drivers []DriverInfo, opt map[string]Options, pw
 			mu.Lock()
 			resp[k] = rr
 			mu.Unlock()
+			if opt.ImageIDFile != "" {
+				return ioutil.WriteFile(opt.ImageIDFile, []byte(rr.ExporterResponse["containerimage.digest"]), 0644)
```

Comment:
Add a switch between manifest digest and this. If manifest digest exists use that.